mimic/test/test_nova.py

```diff
@@ -35,20 +34,67 @@ def status_of_server(test_case, server_id):
     return get_server_response_body['server']['status']
 
 
-def quick_create_server(helper, region="ORD"):
+def server_args(name=None, imageRef=None, flavorRef=None,
```

Comment:
I'd like to avoid polluting the namespace with too many extra names of helper functions (we already have a ton that we need, so extras are bad).  It seems like every single invocation of `server_args` is passed as the second argument to `create_server`, and no calls to `create_server` are ever made without also calling `server_args`.  Whenever I see two functions being used together like that, it seems to me that they're really just two parts of the same whole and they ought to be presented as such.

In this case, that would mean `create_server` should take all the keyword args of `server_args` and do the serialization itself.  If a case were to arise where we needed to provide a bad request body (for example to test 400 replies) then we could add an additional `raw_body` kwarg to `create_server` which would take precedence.

What do you think?